### PR TITLE
Added Circle-CI.

### LINKED ISSUE #257: Add CircleCI to the 2020 Hacktoberfest Swag List
A good first issue for people celebrating Hacktoberfest! If you want for this to count towards Hacktoberfest, please submit your PR on October 1st!

Per [here](https://hacktoberfest.circleci.com/), [CircleCI](https://circleci.com/) is participating in Hacktoberfest 2020 (which they dub Orbtoberfest) by giving away their own swag.

Please add CircleCI to the 2020 list following the formatting in the [CONTRIBUTING.md](https://github.com/crweiner/hacktoberfest-swag-list/blob/master/CONTRIBUTING.md) file.

You will need to locate the type of swag being offered and a link to their signup form (or some info on how they are tracking contributors).

Be sure to register for Hacktoberest first by [clicking here](https://hacktoberfest.digitalocean.com/).

### PR DESCRIPTION
**Pull request by DaffyTheDuck**
Thanks for contributing to the [Hacktoberfest Swag List](https://hacktoberfestswaglist.com/) :smiley: :tada:! Before submitting your pull request, please check off as many of the items below as you can:

1. [x] I have read the [Contributing.md](../CONTRIBUTING.md) file and formatted this PR correctly
2. [x] I'm not adding a company from the blocklist
3. [x] I make sure to fix things promptly if an error or suggestion comes up

Thanks and Happy Hacktoberfest! :tada:
Tagging @crweiner to take a look. :eyes:

Closes #257

### CODE REVIEW

**Comment by crweiner** [comment]
Hi @DaffyTheDuck! Can you please work on the issues flagged by Code Climate on this and your other PRs?

Thanks and let me know if you have any questions! 👍

**Comment by DaffyTheDuck** [comment]
yes fixing those!

**Comment by DaffyTheDuck** [comment]
Closes #257

**Comment by hugovk** [comment]
> Closes #257

Tip: put this in the first message in the PR, or it won't take affect automatically.

**Comment by DaffyTheDuck** [comment]
> > Closes #257
> 
> Tip: put this in the first message in the PR, or it won't take affect automatically.

:sweat_smile: I know, but I thought changing default message won't be good :sweat_smile:

**Comment by hugovk** [comment]
I think adding it and keeping the rest of the template (like you just did :) is fine! 👍

File: README.md [hunk 1]
```diff
@@ -47,6 +47,14 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
 - **How to sign up**: First you have submit two PRs then let us know you’ve completed the challenge by filling out the form on our [official Hacktoberfest page](https://camunda.com/hacktoberfest2020/) and we will send you a e-mail. The repos are: [Camunda BPM](https://github.com/camunda/), [bpmn.io](https://github.com/bpmn-io/), and [Zeebe-io](https://github.com/zeebe-io/).
 - **Notes**: Check out Camunda [official Hacktoberfest page](https://camunda.com/hacktoberfest2020/) for more info & how to get the T-shirt.
 
+#### **Circle-CI**
+
+- **Swag**: T-shirt, stickers, Limited edition orb magic pin(special case)
+- **Requirements**: 4+ merged or accpeted PRs
```
Comment (crweiner) [inline README.md:53]: Can you please split this into the different levels? 1-3 PRs, 4+ PRs, and Orb development?

File: README.md [hunk 2]
```diff
@@ -103,6 +111,12 @@ None right now.
 - **Issues**: [#hacktoberfest](https://github.com/search?&q=user%3Aappwrite+label%3Ahacktoberfest+type:issue&state=open&type=Issues) or any issue you like to help with.
 - **Notes**: Follow [Appwrite blog](https://medium.com/appwrite-io) and [Discord community](https://appwrite.io/discord) for more information and help.
 
+#### **Circle-CI** (T-shirt, stickers, limited edition orb magic pin)
```
Comment (crweiner) [inline README.md:114]: Please change the title to only show the stickers earned at this level.

File: README.md [hunk 3]
```diff
@@ -142,6 +156,12 @@ None right now.
 - **How to sign up**: First you have submit two PRs then let us know you’ve completed the challenge by filling out the form on our [official Hacktoberfest page](https://camunda.com/hacktoberfest2020/) and we will send you a e-mail. The repos are: [Camunda BPM](https://github.com/camunda/), [bpmn.io](https://github.com/bpmn-io/), and [Zeebe-io](https://github.com/zeebe-io/).
 - **Notes**: Check out Camunda [official Hacktoberfest page](https://camunda.com/hacktoberfest2020/) for more info & how to get the T-shirt.
 
+#### **Circle-CI** (T-shirt, stickers, limited edition orb magic pin)
+
+- **Requirements**: Create 4 PRs on any of [Circle-CI Repos](https://github.com/circleci) as well as make sure you fill the form [here](https://circleci-community.typeform.com/to/mErGL0nv) after making them.
+- **Swag**: Sticker pack and t-shirt
```
Comment (crweiner) [inline README.md:162]: Please let the great people know how they can earn the Orb pin too! 😄

File: README.md [hunk 4]
```diff
@@ -47,6 +47,14 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
 - **How to sign up**: First you have submit two PRs then let us know you’ve completed the challenge by filling out the form on our [official Hacktoberfest page](https://camunda.com/hacktoberfest2020/) and we will send you a e-mail. The repos are: [Camunda BPM](https://github.com/camunda/), [bpmn.io](https://github.com/bpmn-io/), and [Zeebe-io](https://github.com/zeebe-io/).
 - **Notes**: Check out Camunda [official Hacktoberfest page](https://camunda.com/hacktoberfest2020/) for more info & how to get the T-shirt.
 
+#### **Circle-CI**
+
+- **Swag**: T-shirt, stickers, Limited edition orb magic pin(special case)
+- **Requirements**: 4+ merged or accpeted PRs
+- **How to sign up**: First you've to submit the PRs as per reuirements and once you've completed your PRs, submit the links using [this form.](https://circleci-community.typeform.com/to/mErGL0nv).
+- **Notes**: Check out the [community forum](https://discuss.circleci.com/)for ideas, FAQs, and to ask any questions. You can read more [here](https://hacktoberfest.circleci.com/).
+- **Special Case**: You've to complete the challenge 2 which is mentioned [here](https://hacktoberfest.circleci.com/).
```
Comment (crweiner) [inline README.md:56]: You can remove this line, please. 👍

**Review by crweiner — COMMENTED** [review]
Hi @DaffyTheDuck! Thanks for your contribution! Can you make a few changes for me, please? There are other PRs out there to also add CircleCI so I'll be picking and choosing the best parts of each. 

The merge conflict should be simple to fix too, please. Thank you! 👍

**Comment by DaffyTheDuck** [comment]
Yeah! sure!
Fixing those

File: README.md (hunk 4 above)
Comment (DaffyTheDuck) [reply README.md:56]: The last one right?

**Comment by DaffyTheDuck** [comment]
Fixed the conflict, any other changes?

File: README.md (hunk 3 above)
Comment (DaffyTheDuck) [reply README.md]: Or I have to mention exact what they need to do for challenge-2 ?

**Comment by DaffyTheDuck** [comment]
Hello! @crweiner should I squash the commit's as well?

**Comment by crweiner** [comment]
Hi @DaffyTheDuck! Any particular reason why you closed this? I finally have a moment to review all of the PRs and yours was towards the top of my stack. 😄

**Comment by DaffyTheDuck** [comment]
Oh no It was an accident

**Comment by DaffyTheDuck** [comment]
I was waiting for your review :(

**Comment by crweiner** [comment]
No worries! Honest mistakes happen. I appreciate you taking the time to contribute to the Swag List!! 👍 😄

**Pull request closed without merging**